### ISSUE TITLE
Link to user page from category manager list

### ISSUE THREAD

**Issue by OmeGak** [indico/indico#2005]
Related SNOW ticket `CHG33945`.

Currently, there is no way to contact category managers from the category display view. The only way at the moment is looking for the name in CERN Phonebook, which is not always applicable (external accounts, other Indico instances, etc.). 

Feedback suggested simply displaying the primary email address, but we should rather not disclose it. The solution would be linking to a *yet-to-be-implemented* user profile page and from there, based on user privileges, allow checking the email address.

**Comment by ThiefMaster** [comment]
Wouldn't work for groups though.

**Comment by pferreir** [comment]
> At some point such a profile page could actually contain a way to send a message to another user.

Wouldn't work for groups either. Unless we allow the sending of messages to all group members, which I doubt that would be a good idea.

**Comment by ThiefMaster** [comment]
Yeah, it'd be only for users. But being able to quickly contact an Indico user doesn't sound like such a bad idea.

**Comment by pferreir** [comment]
No, not at all. I totally agree with that.

**Comment by OmeGak** [comment]
> all software expands until it includes messaging

**Comment by pferreir** [comment]
> all software expands until it includes messaging

We'll have emojis, stickers and embedded YouTube videos :dancer:
